assets/src/blocks/amp-story-title/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Much of this block is taken from the Core Heading block.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { RichText } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export const name = 'amp/amp-story-title';
+
+export const settings = {
+	title: __( 'AMP Title', 'amp' ),
```

Comment:
```suggestion
	title: __( 'Story Title', 'amp' ),
```

Reply:
Don't accept this suggestion unless you want the CLA bot to start ❌'ing your PR 😏